saadmob/src/main/java/tv/superawesome/plugins/publisher/admob/SAAdMobInterstitialCustomEvent.kt

```diff
@@ -50,7 +50,7 @@ class SAAdMobInterstitialCustomEvent : CustomEventInterstitial {
                         ""
                     )
                 )
-                SAEvent.adAlreadyLoaded, SAEvent.adEnded -> {}
+                SAEvent.adAlreadyLoaded, SAEvent.adEnded, SAEvent.adPaused, SAEvent.adPlaying -> Unit // Unused
```

Comment:
```suggestion
                else -> Unit // Unused
```

You will need to reorder the cases here too.